website/content/docs/consul-vs-other/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+layout: docs
+page_title: Consul versus other service networking software
+description: >-
+  Consul is service networking platform that addresses four core use cases relating to their network services: centralized service discovery, zero trust networking with service mesh, automating network infrastructure, and controlling access to mesh services with the Consul API Gateway. This doc will compare Consul with other software that provide similar capabilities with one or more of the core use cases.
```

Comment:
```suggestion
  Consul is a service networking platform that centralizes service discovery, enables zero trust networking with service mesh, automates network infrastructure, and controls access to mesh services with the Consul API Gateway. Compare Consul with other software that provide similar capabilities with one or more of the core use cases.
```
I'm not sure what the target SEO phrase is, but I might even cut ". . . is a service networking platform that . . ." to get it closer to the 180 word cut-off